### ISSUE TITLE
Error debugging

### ISSUE THREAD

**Issue by cristianmad** [gloveboxes/dotNET-Core-IoT-Raspberry-Pi-Linux.-How-to-Build-Deploy-and-Debug#1]
Hi, 
Great guide, thanks for putting it together!

I'm running into some issues when launching and attempting to debug. Everything is OK up until the point where it attempts to attach to the executing application in order to debug.
Would you have any idea what might be wrong?

Starting: "/usr/bin/ssh" pi@raspberrypi.local "~/vsdbg/vsdbg --interpreter=vscode"
Error from pipe program 'ssh': bash: /home/pi/vsdbg/vsdbg: No such file or directory

It looks like it is not able to find the debugger, however, unless I missed something, I did not see a dedicated step for that.
I'm using a Mac & RaspberryPi3.
I should add that the build output is being deployed on the RaspberryPi.

Thanks!

UPDATE: I found the solution. Install the remote debug tools as described here:
https://www.hanselman.com/blog/RemoteDebuggingWithVSCodeOnWindowsToARaspberryPiUsingNETCoreOnARM.aspx
Note the the article is using a Windows system. I'm using a Mac, however the solution is the same.

Basically simply run the following command on the RaspberryPi:
curl -sSL https://aka.ms/getvsdbgsh | /bin/sh /dev/stdin -v latest -l ~/vsdbg

**Comment by gloveboxes** [comment]
Hey great that you enjoyed the guide and you figured out the issue. In the guide, I wrote

Configure Connection to your Raspberry Pi
The following creates a new SSH key, copies the public key to the Raspberry Pi, and then installs the Visual Studio Debugger on the Raspberry Pi. Take the default options.

From Linux and macOS
Open a new Terminal, and copy and paste the following command.

ssh-keygen -t rsa && ssh-copy-id pi@raspberrypi.local && \
ssh pi@raspberry.local "curl -sSL https://aka.ms/getvsdbgsh | bash /dev/stdin -r linux-arm -v latest -l ~/vsdbg"

Did you miss this step or did it fail?

Cheers Dave

**Comment by cristianmad** [comment]
Hi Dave, 
Thanks for the reply!
I did see those (and executed) when performing the initial setup. I believe that when copy/pasting the two lines, only the first one may have executed (just guessing). I do not remember seeing any kind of errors during that setup. It is possible I may have done something wrong :).


Thanks, 
Cristian
